docs/source/notebooks/Embedding_nets.ipynb

```diff
@@ -0,0 +1,420 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we combine our synthetic data into a dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = [photometry,\n",
+    "           # spec0, #Fitting spectra is slow because it needs to do a lot of resampling\n",
+    "           spec1   # As a result, we're leaving some of them out\n",
+    "           ]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Defining an embedding network and doing inference\n",
+    "\n",
+    "Now that we have our model and (synthetic) data, the last thing we need before we can do inference is our embedding. \n",
+    "SBI is all uses torch, so we need to define our embedding as a torch neural network - or at least one that torch understands. \n",
+    "We *could* do that from scratch, with something like the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torch import Tensor, nn\n",
+    "\n",
+    "\n",
+    "class FullyCOnnectedNetwork(nn.Module):\n",
+    "    def __init__(\n",
+    "        self,\n",
+    "        input_dim,\n",
+    "        output_dim = 20,\n",
+    "        num_layers = 3,\n",
+    "        num_hidden_units = 100,\n",
+    "    ):\n",
+    "        super().__init__()\n",
+    "        layers = [nn.Linear(input_dim, num_hiddens), nn.ReLU()]\n",
+    "        for _ in range(num_layers - 2):\n",
+    "            layers.append(nn.Linear(num_hiddens, num_hidden_units))\n",
+    "            layers.append(nn.ReLU())\n",
+    "        layers.append(nn.Linear(num_hiddens, output_dim))\n",
+    "        layers.append(nn.ReLU())\n",
+    "        self.net = nn.Sequential(*layers)\n",
+    "\n",
+    "    def forward(self, x: Tensor) -> Tensor:\n",
+    "        return self.net(x)\n",
+    "\n",
+    "embedding = FullyCOnnectedNetwork(np.sum([data.wavelength.shape[0] for data in dataset]))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can find some further information on defining embedding networks in the SBI documentation at https://www.mackelab.org/sbi/tutorial/05_embedding_net/.\n",
+    "\n",
+    "However, SBI also packages a small selection of pre-defined networks that you can use, and we can specify the same network by just importing one of those and passing it the same setup, for example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sbi.neural_nets.embedding_nets import FCEmbedding\n",
+    "\n",
+    "embedding = FCEmbedding(np.sum([data.wavelength.shape[0] for data in dataset]), output_dim=20, num_layers=3, num_hidden_units=100)"
```

Comment:
`num_hidden_units` must be changed to `num_hiddens` everywhere, as this is the correct argument name to pass to FCEmbedding.

Reply:
Fixed